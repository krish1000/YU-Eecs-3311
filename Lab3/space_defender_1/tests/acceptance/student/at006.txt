play(5,10,12,3)
fire
fire
move(C,5) -- move(C,10) cause violation
fire
fire
undo
undo
undo
undo
undo
undo