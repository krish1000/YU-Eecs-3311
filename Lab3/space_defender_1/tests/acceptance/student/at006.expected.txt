  Welcome to Space Defender Version 1.
->play(5,10,5,2)
  state:1.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:2.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:3.0, ok
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  *  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:4.0, ok
  A projectile moves: [C,4] -> [C,6]
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  *  _  *  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:5.0, ok
  A projectile moves: [C,6] -> [C,8]
  A projectile moves: [C,4] -> [C,6]
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  *  _  *  _  *  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->undo
  state:4.0, ok
  A projectile moves: [C,4] -> [C,6]
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  *  _  *  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->undo
  state:3.0, ok
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  *  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->undo
  state:2.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->undo
  state:1.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->undo
  state:1.1, error
  Nothing left to undo.